ts/services/NewReportsService.ts

```diff
@@ -0,0 +1,75 @@
+import * as moment from 'moment';
+import * as caporal from 'caporal';
+import * as request from 'request-promise';
+
+class NewReportsService {
+
+  public async ec2rightsizing(config, logger): Promise<void> {
+    logger.debug('\nListing the latest EC2 Right Sizing report');
```

Comment:
Listing the report? "Listing" implies multiple reports, "report" is singular. How many reports are returned by this?

Reply:
changed to 'querying'